vmdb/app/models/ems_kubernetes.rb

```diff
@@ -3,6 +3,7 @@ class EmsKubernetes < EmsContainer
   has_many :container_groups,                     :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_replication_controllers,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_namespaces,                 :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
I merged #2611, so we this should be discussed with respect to that being already in.